### DISCUSSION TITLE
Python installation can't find readline and bzip2

### DISCUSSION THREAD

**Discussion by penyuan** [pyenv/pyenv#2205] [Q&A] [answered]
I've installed Pyenv on my RHEL 8.5 GNU/Linux system with Homebrew: 

```
brew install pyenv
```

Then I tried to install Python 3.10.1 with Pyenv but got the following `WARNING`s: 

```
pyenv install 3.10.1
Downloading Python-3.10.1.tar.xz...
-> https://www.python.org/ftp/python/3.10.1/Python-3.10.1.tar.xz
Installing Python-3.10.1...
python-build: use readline from homebrew
python-build: use zlib from homebrew
WARNING: The Python bz2 extension was not compiled. Missing the bzip2 lib?
WARNING: The Python readline extension was not compiled. Missing the GNU readline lib?
Installed Python-3.10.1 to /home/penyuan/.pyenv/versions/3.10.1
```

Looks like it's having trouble finding "readline" and "zlib", but I've already installed the `readline`, `zlib`, `lzlib`, `bzip2`, and `lbzip2` packages from Homebrew. I also have all of them installed via my host system's package manager, too.

So why isn't Pyenv's Python installation process finding the libraries they need?

Thanks!

**Comment by tedmiston** [comment, marked as answer]
This is an ongoing problem with pyenv, see https://github.com/pyenv/pyenv/issues/1479 for more info.

The short-term workaround is to temporarily uninstall GNU readline from brew per this comment https://github.com/pyenv/pyenv/issues/1479#issuecomment-587044749.

**Reply by penyuan** [reply]
Thank you, I have subscribed to that issue. Hope this will be fixed soon.
